spec/requests/admin/notify_downloads_spec.rb

```diff
@@ -0,0 +1,113 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Notify Downloads', type: :request do
+  include SingleSignOnHelpers
+
+  before do
+    stub_govuk_bank_holidays_request
+    mock_sso_with(email: 'admin@example.com')
+    get '/auth/google_oauth2/callback'
+  end
+
+  describe '#index' do
+    it 'lists the available Notify downloads' do
+      get admin_notify_downloads_path
+
+      expect(response).to be_successful
+      expect(response.body).to include('Notify downloads')
+      expect(response.body).to include('Management information is overdue')
+      expect(response.body).to include(admin_notify_download_path(:overdue))
```

Comment:
Same for here, why not `have_link`?

Reply:
as above, it's the difference between a request spec and a feature spec.